setup.py

```diff
@@ -77,7 +57,17 @@ def get_packages(package):
         'Programming Language :: Python :: 3.4',
         'Topic :: Utilities',
     ],
+    # install_requires=['cryptography'],
     test_suite='tests',
-    tests_require=['pytest', 'pytest-cov'],
-    cmdclass={'test': PyTest},
```

Comment:
Its a setuptools convention: http://pythonhosted.org//setuptools/setuptools.html#test-build-package-and-run-a-unittest-suite

i.e. Anytime you see a setup.py, you should be able to run `setup.py test` and run the tests for the package.